routes.py

```diff
@@ -1,10 +1,25 @@
 from fastapi import APIRouter,Depends,HTTPException
-from services import initial_service
+from models import Blog
+from services import create_blog, initial_service, update_blog
 
 
 
 router=APIRouter()
 
 @router.get('/')
 async def getInitial():
-    return initial_service()
\ No newline at end of file
+    return initial_service()
+
+@router.post('/createblog', response_model=Blog)
+async def createBlog(blog: Blog):
+    response = await create_blog(blog)
+    if response:
```

Comment:
no need to handle the exceptions again on here, because u already handled it in service.py